packages/integrations/cloudflare/README.md

```diff
@@ -147,38 +146,80 @@ If you want to use the automatic `_routes.json` generation, but want to exclude
-+       },
+        routes: {
+            strategy: 'include',
+            include: ['/users/*'], // handled by custom function: functions/users/[id].js
+            exclude: ['/users/faq'], // handled by static page: pages/users/faq.astro
+        },
     }),
 });
 ```
 
-## Enabling Preview
+### `wasmModuleImports`
+
+`wasmModuleImports: boolean`
 
-In order for preview to work you must install `wrangler`
+default: `false`
 
-```sh
-pnpm install wrangler --save-dev
+Whether or not to import `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration) using the `.wasm?module` import syntax.
+
+Add `wasmModuleImports: true` to `astro.config.mjs` to enable this functionality in both the Cloudflare build and the Astro dev server. [Read more](#use-wasm-modules)
+
+```js ins={7}
+// astro.config.mjs
+import {defineConfig} from "astro/config";
+import cloudflare from '@astrojs/cloudflare';
+
+export default defineConfig({
+    adapter: cloudflare({
+       wasmModuleImports: true
+    }),
+    output: 'server'
+})
 ```
 
-It's then possible to update the preview script in your `package.json` to `"preview": "wrangler pages dev ./dist"`. This will allow you to run your entire application locally with [Wrangler](https://github.com/cloudflare/wrangler2), which supports secrets, environment variables, KV namespaces, Durable Objects and [all other supported Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/#adding-bindings).
+### `runtime`
+
+`runtime: "off" | "local" | "remote"`
+
+default `"off"`
+
+Determines whether and how the Cloudflare Runtime is added to `astro dev`. The Cloudflare Runtime includes [Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/bindings), [environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables), and the [cf object](https://developers.cloudflare.com/workers/runtime-apis/request/#incomingrequestcfproperties). [Read more](#access-to-the-cloudflare-runtime)
+
+- `local`: uses bindings mocking and locally static placeholdes
+- `remote`: uses remote bindings and a live fetched cf object
+- `off`: no access to the Cloudflare runtime using `astro dev`. Alternativly use [Preview with Wrangler](#preview-with-wrangler)
+
+```js ins={8}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import cloudflare from '@astrojs/cloudflare';
 
-## Access to the Cloudflare runtime
+export default defineConfig({
+  output: 'server',
+  adapter: cloudflare({
+    runtime: 'local',
+  }),
+});
+```
 
-You can access all the Cloudflare bindings and environment variables from Astro components and API routes through `Astro.locals`.
+## Cloudflare runtime
+
+You can access the runtime from Astro components and API routes through `Astro.locals`.
 
 If you're inside an `.astro` file, you access the runtime using the `Astro.locals` global:
```

Comment:
```suggestion
You can access the runtime from Astro components and API routes through `Astro.locals` inside any .astro` file.
```
I think these two sentences basically said the same thing? Can they be combined into one?